packages/esm-patient-appointments-app/src/appointments/appointments-overview.component.tsx

```diff
@@ -42,7 +42,7 @@ const AppointmentsOverview: React.FC<AppointmentOverviewProps> = ({ patientUuid
 
       return () => abortController.abort();
     }
-  }, [patientUuid, startDate]);
```

Comment:
Why don't we "fix" start date instead? I think right now every re-render would update the start date. Shouldn't it be a constant instead?

Reply:
We can make it a state instead, it won't update on every re-render. What you say?

Reply:
Yeah that would be a suitable workaround (actually it can also just be a memo - so it does not matter if you use `useState` and `useMemo` except that the latter would not require array destructing and thus is a bit simpler to look out / understand).

However, what happens then if we re-mount? My question is - shouldn't the appointment start be the actual appointment start date?

Reply:
I'd say let's not discuss this.

Let's just make `const startDate = useMemo(getStartDate, []);` where

```
function getStartDate() {
  return dayjs().format();
}
```

is a global utility.

Reply:
Sure! Thanks @FlorianRappl ! I'll do it and pushing it asap.